client/assets/stylesheets/style.scss

```diff
@@ -19,3 +19,6 @@
 
 // Overrides for @wordpress/components
 @import "./wp-components-overrides";
+
+// Overrides for @wordpress/base-styles
+@import "./wp-base-styles-overrides";
```

Comment:
Should the base style overrides be applied before the component overrides?

Reply:
Maybe? I haven't seen any adverse affects, but that's probably because the base overrides are all very targeted and don't affect the components at this point. I'll swap them in case that changes in future.